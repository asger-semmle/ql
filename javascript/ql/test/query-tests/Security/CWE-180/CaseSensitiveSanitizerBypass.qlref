Security/CWE-180/CaseSensitiveSanitizerBypass.ql
