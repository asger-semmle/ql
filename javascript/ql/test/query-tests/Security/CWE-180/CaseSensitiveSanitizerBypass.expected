nodes
| index.js:1:30:1:33 | html |
| index.js:1:30:1:33 | html |
| index.js:4:13:4:16 | html |
| index.js:4:13:4:29 | html.substring(i) |
| index.js:4:13:4:29 | html.substring(i) |
| index.js:18:31:18:34 | html |
| index.js:18:31:18:34 | html |
| index.js:19:14:19:16 | tag |
| index.js:19:21:19:37 | getHtmlTags(html) |
| index.js:19:33:19:36 | html |
| index.js:20:25:20:27 | tag |
| index.js:29:22:29:24 | tag |
| index.js:30:33:30:35 | tag |
| index.js:30:33:30:35 | tag |
| index.js:50:30:50:32 | url |
| index.js:50:30:50:32 | url |
| index.js:51:9:51:11 | url |
| index.js:51:9:51:11 | url |
| index.js:66:30:66:32 | url |
| index.js:66:30:66:32 | url |
| index.js:67:38:67:40 | url |
| index.js:67:38:67:49 | url.protocol |
| index.js:67:38:67:49 | url.protocol |
edges
| index.js:1:30:1:33 | html | index.js:4:13:4:16 | html |
| index.js:1:30:1:33 | html | index.js:4:13:4:16 | html |
| index.js:4:13:4:16 | html | index.js:4:13:4:29 | html.substring(i) |
| index.js:4:13:4:16 | html | index.js:4:13:4:29 | html.substring(i) |
| index.js:18:31:18:34 | html | index.js:19:33:19:36 | html |
| index.js:18:31:18:34 | html | index.js:19:33:19:36 | html |
| index.js:19:14:19:16 | tag | index.js:20:25:20:27 | tag |
| index.js:19:21:19:37 | getHtmlTags(html) | index.js:19:14:19:16 | tag |
| index.js:19:33:19:36 | html | index.js:19:14:19:16 | tag |
| index.js:19:33:19:36 | html | index.js:19:21:19:37 | getHtmlTags(html) |
| index.js:20:25:20:27 | tag | index.js:29:22:29:24 | tag |
| index.js:29:22:29:24 | tag | index.js:30:33:30:35 | tag |
| index.js:29:22:29:24 | tag | index.js:30:33:30:35 | tag |
| index.js:50:30:50:32 | url | index.js:51:9:51:11 | url |
| index.js:50:30:50:32 | url | index.js:51:9:51:11 | url |
| index.js:50:30:50:32 | url | index.js:51:9:51:11 | url |
| index.js:50:30:50:32 | url | index.js:51:9:51:11 | url |
| index.js:66:30:66:32 | url | index.js:67:38:67:40 | url |
| index.js:66:30:66:32 | url | index.js:67:38:67:40 | url |
| index.js:67:38:67:40 | url | index.js:67:38:67:49 | url.protocol |
| index.js:67:38:67:40 | url | index.js:67:38:67:49 | url.protocol |
#select
| index.js:4:13:4:29 | html.substring(i) | index.js:1:30:1:33 | html | index.js:4:13:4:29 | html.substring(i) | This check can be bypasssed by mixed-case inputs from $@. | index.js:1:30:1:33 | html | here |
| index.js:30:33:30:35 | tag | index.js:18:31:18:34 | html | index.js:30:33:30:35 | tag | This check can be bypasssed by mixed-case inputs from $@. | index.js:18:31:18:34 | html | here |
| index.js:51:9:51:11 | url | index.js:50:30:50:32 | url | index.js:51:9:51:11 | url | This check can be bypasssed by mixed-case inputs from $@. | index.js:50:30:50:32 | url | here |
| index.js:67:38:67:49 | url.protocol | index.js:66:30:66:32 | url | index.js:67:38:67:49 | url.protocol | This check can be bypasssed by mixed-case inputs from $@. | index.js:66:30:66:32 | url | here |
